templates/components/products/card.html

```diff
@@ -16,14 +16,21 @@
                 </div>
             {{/if}}
         {{/or}}
-        <a href="{{url}}">
+        {{#if id '>' 0}}
```

Comment:
This id is product id which is the index key in product db table. id is a positive number and uniq for real products. For sample products we assign negative numbers to id. 

On homepage, if there is no inventory products we display sample products with no URL and no Quick View.